src/AppBundle/Service/EventProcessor.php

```diff
@@ -348,18 +348,19 @@ private function setContributionsWikipedias(EventWiki $wiki, EventWikiRepository
         $wiki->setPagesEdited($pageIdsEdited);
 
         $pageIds = array_merge($pageIdsCreated, $pageIdsEdited);
-        $ret = $this->eventRepo->getEditStats($dbName, $pageIds, $start, $end, $usernames);
-
         $diff = $ewRepo->getBytesChanged($this->event, $dbName, $pageIds, $usernames);
 
-        $this->pagesCreated += $ret['created'];
-        $this->pagesImproved += $ret['edited'];
-        $this->edits += $ret['edits'];
+        $totalCreated = count($pageIdsCreated);
+        $totalEdited = count($pageIdsEdited);
+        $total = count($pageIds);
+        $this->pagesCreated += $totalCreated;
+        $this->pagesImproved += $totalEdited;
+        $this->edits += $total;
```

Comment:
`pagesCreated` is the number of unique pages improved, `pagesCreated` is the number of unique pages created. Neither is a count of the edits made to those pages, right? I don't think we're querying for that anymore (it was one of the things `getEditStats()` did).

Reply:
doh! Yes, you're quite right. :) I'll fix it up.